packages/now-cli/src/index.js

```diff
@@ -515,17 +513,44 @@ const main = async argv_ => {
     );
   }
 
+  const {
+    authConfig: { token },
+  } = ctx;
+  const client = new Client({ apiUrl, token });
+
+  let scope = argv['--scope'] || argv['--team'] || localConfig.scope;
+
+  // overwrite scope with `NOW_ORG_ID` if defined
+  const { NOW_ORG_ID } = process.env;
+  if (NOW_ORG_ID) {
+    const org = await getLinkedOrg(client);
+
+    if (!org) {
+      output.print(
+        `${chalk.red('Error!')} Organization not found (${JSON.stringify({
+          NOW_ORG_ID,
+        })})\n`
+      );
+      return 1;
+    }
+
+    scope = org.slug;
+  }
+
+  // use local `.now` scope if not explicitly specified
+  if (!scope) {
+    const org = await getLinkedOrg(client);
+    scope = org ? org.slug : undefined;
+  }
+
+  const targetCommand = commands.get(subcommand);
+
   if (
     typeof scope === 'string' &&
     targetCommand !== 'login' &&
     targetCommand !== 'dev' &&
     !(targetCommand === 'teams' && argv._[3] !== 'invite')
   ) {
-    const {
-      authConfig: { token },
-    } = ctx;
-    const client = new Client({ apiUrl, token });
-
```

Comment:
We used to create the `client` only in here to make sure we don't make any requests to the API for certain commands when no token is available or required, I think we should keep it that way and only create a client when checking NOW_ORG_ID, additionally we could also fail right away when we don't have a token and need to check the org

Reply:
Tackled in d145e6164074fe2442178cd8fafbeb225c978b9c